package.json

```diff
@@ -448,4 +460,4 @@
         "webpack": "4.36.1",
         "webpack-cli": "3.3.6"
     }
-}
+}
```

Comment:
You should change your vs code settings to add a newline to the end of every file, and re-save all these